packages/main/src/Input.ts

```diff
@@ -783,6 +783,13 @@ class Input extends UI5Element implements SuggestionComponent, IFormInputElement
 		}
 
 		if (isEnter(e)) {
+			const isValueChanged = this.previousValue === this.getInputDOMRefSync()!.value;
+			this._enterKeyDown = true;
+
+			if (isValueChanged && this._internals.form) {
+				submitForm(this);
```

Comment:
Why? This prevents firing of the change event. Defeats the purpose of the issu.

Reply:
Then again, looking at the implementation of `_handleEnter`: You also have logic for selecting suggestion items via "Enter". 

Wouldn't that also (falsely) trigger the submit?